### PR TITLE
Tuning Elasticsearch--SME Review, LRDOCS-4744

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-4744

### CODE REVIEW

**Comment by rbohl** [comment]
@arboliveira, please review. Thanks!

File: discover/deployment/articles/01-installing-search-engine/05-tuning.markdown [hunk 1]
```diff
@@ -30,36 +30,19 @@ default Linux value is 1024). Since Elasticsearch uses NioFS and MMapFS, ensure
...
+to design the optimally scaled and tuned cluster. Just note that the number of
+shards and replicas are actually configured in the Liferay Connector to
+Elasticsearch module, using these settings:
+
+- `indexNumberOfReplicas` corresponds to Elasticsearch's `number_of_replicas`
+    property.
+- `indexNumberOfShards` corresponds to Elasticsearch's `number_of_shards`
+    property.
+
+Tune, scale, and prosper.
```
Comment (arboliveira) [inline discover/deployment/articles/01-installing-search-engine/05-tuning.markdown]: Proper scaling and tuning of an Elasticsearch cluster depends first and foremost on the specific usage patterns for the indexes it holds. With @product@ being extremely flexible as a development platform, no two applications will index and search for data in exactly the same way. We recommend reading the [definitive Elasticsearch guide](https://www.elastic.co/guide/en/elasticsearch/guide/master/distributed-cluster.html), understanding the differences between [indexing intensive applications](https://www.elastic.co/guide/en/elasticsearch/reference/master/tune-for-indexing-speed.html) and [search intensive applications](https://www.elastic.co/guide/en/elasticsearch/reference/master/tune-for-search-speed.html), to anticipate expected usage patterns for your @product@ indexes and design the optimally scaled and tuned cluster.

As you decide how many shards and replicas for @product@ indexes are best suited for your applications, these numbers can be configured in the Liferay Connector to Elasticsearch module, using the settings:

- `indexNumberOfReplicas` corresponds to Elasticsearch's `number_of_replicas`
    property.
- `indexNumberOfShards` corresponds to Elasticsearch's `number_of_shards`
    property.

Tune, scale, and prosper.

Comment (rbohl) [reply discover/deployment/articles/01-installing-search-engine/05-tuning.markdown]: Thanks @arboliveira . Review edits applied here: https://github.com/rbohl/liferay-docs/pull/85/commits/435d8f54525a7e8d5f227aef0558056c9c900b06

Is this ready for publication?

**Comment by rbohl** [comment]
@arboliveira I added a commit to incorporate your edits.
https://github.com/rbohl/liferay-docs/pull/85/commits/435d8f54525a7e8d5f227aef0558056c9c900b06
Let me know if you think this is ready to go. Thanks!

**Comment by arboliveira** [comment]
LGTM! :shipit:

**Comment by rbohl** [comment]
Sent for publication: https://github.com/sez11a/liferay-docs/pull/3495

**Pull request closed without merging**